patches/moveUserOptionsToHeader/styles.css

```diff
@@ -0,0 +1,122 @@
+@media screen and (max-width: 1023px) {
+    .user__section,
+    div:has(.user__links__conatainer) {
+        display: none !important;
+    }
+
+    .app.hideAside aside {
+        display: none !important;
+    }
+
+    .header_logo_tools_user-wrapper img {
+        width: 45px;
+        height: 45px;
+        margin-right: 10px;
+        cursor: pointer;
+    }
+
+    .modal-background {
+        display: none;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100vw;
+        height: 100%;
+        background-color: rgba(0, 0, 0, 0.5);
+        z-index: 9999;
+    }
+
+    .modal-background.active {
+        display: block !important;
+    }
+
+    .modal-user {
+        display: none;
+        top: 50%;
+        left: 50%;
+        position: absolute;
+        overflow: hidden;
+        transform: translate(-50%, -50%);
+
+        width: 80vw;
+        max-width: 400px;
+        height: min-content !important;
+        max-height: min-content !important;
+        border-radius: 10px;
+        background-color: #fff;
+        z-index: 10000;
+        padding: 10px;
+    }
+
+    .modal-user.active {
+        display: block !important;
+    }
+
+    .modal-data {
+        display: flex;
+        align-items: center;
+        gap: 10px;
+        padding: 10px 20px;
+        margin-bottom: 10px;
+    }
+
+    .modal-name {
+        display: flex;
+        flex-direction: column;
+        padding-left: none;
+        gap: 5px;
+    }
+
+    .modal-link-container,
+    .modal-back-container {
+        display: flex;
+        align-items: center;
+        padding: 10px 20px;
+        cursor: pointer;
+    }
+
+    .modal-back-container {
+        justify-content: flex-end;
+    }
+
+    .modal-link-container span {
+        padding: 0;
+        font-size: 1.2rem;
+        font-weight: 500;
+        text-decoration: none;
+        color: #3d4455;
+    }
+
+    .modal-cancel {
+        text-align: right;
+        color: #e33f4a;
+        font-size: 1.2rem;
+        font-weight: 500;
+        cursor: pointer;
+    }
+
+    .header__logo-product > img {
+        width: auto !important;
+    }
+
+    header.app__header {
+        height: 60px;
+    }
+
+    body.blue .user-avatar {
+        background: #38817f !important;

```

Comment:
![opera_MuXJ8p2wOT](https://github.com/user-attachments/assets/e5ad326f-b2d1-4edf-83dc-e5ee35e6dde4)

Text on avatar should be white